README.md

```diff
@@ -45,38 +45,149 @@ Want to setup this stack ?
+- :iphone: [PWA](https://www.npmjs.com/package/@vue/cli-plugin-pwa) : progressive web app support
+- :lipstick: [Prettier](https://prettier.io/) : code formating rules
+- :rotating_light: [Eslint](https://eslint.org/) : control code quality
+- :white_check_mark: [Jest](https://jestjs.io/) : unit testing
+- :white_check_mark: [Cypress](https://www.cypress.io/) : e2e testing
+- :mag: [Vue head](https://github.com/ktquez/vue-head) : meta description per page
+- :page_facing_up: [Optionnal][prerender spa plugin](https://github.com/chrisvfritz/prerender-spa-plugin) : pages prerendering
+- :green_heart: [Optionnal][circleci](https://circleci.com/) : continuous integration/deployment
+- :package: [Optionnal][bundlesize](https://github.com/siddharthkp/bundlesize) : control your js bundles sizes
 
 **App embedded features :**
 
-* :bust_in_silhouette: Google authentication 
-* :mobile_phone_off: Offline support (dynamic & static caching)
-* :new: `New version available` prompt on new app deployments 
-* :heavy_plus_sign: `Add to home screen` prompt for ios & android 
-* :leftwards_arrow_with_hook: Smart redirection for auth protected routes 
-* :sparkles: Products page example to demonstrate app data management with firestore and vuex
-* :muscle: Better PWA support for all browsers with [PWACompat](https://github.com/GoogleChromeLabs/pwacompat)
+- :bust_in_silhouette: Google authentication
+- :mobile_phone_off: Offline support (dynamic & static caching)
+- :new: `New version available` prompt on new app deployments
+- :heavy_plus_sign: `Add to home screen` prompt for ios & android
+- :leftwards_arrow_with_hook: Smart redirection for auth protected routes
+- :sparkles: Products page example to demonstrate app data management with firestore and vuex
+- :muscle: Better PWA support for all browsers with [PWACompat](https://github.com/GoogleChromeLabs/pwacompat)
+
+## Setup
+
+### Pre-requisites
+
+- node@9.3.0+
+- npm@5.5.0+
+
+### Tips
+
+We highly recommand to use [VSCode](https://code.visualstudio.com/) with the following plugins to get a better development experience :
+
+- Prettier
+- Eslint
+- Vetur
+
+**bento-starter** comes with a default code editor config that will automatically be used by vscode. This config is available in `.vscode/settings.json`.
+:::
+
+### Step 1 - Installation
+
+🕙 Estimated time → **20 seconds**
+<br />
+
+```
+git clone https://github.com/kefranabg/bento-starter.git
+cd bento-starter
+cp .env.example .env.local
+npm i
+```
+
+### Step 2 - Firebase configuration
+
+🕙 Estimated time → **3 minutes**
+<br />
+
+- Create a new firebase project with the [firebase console](https://console.firebase.google.com)
+- Once your firebase project is created, add an application by clicking the web button 👉 ![Firebase web app button](/assets/img/firebase-web-btn.jpg) and copy the config object and replace the config variable in `/src/firebase/init.js` in bento-starter project.
```

Comment:
You have to add the `firebase-web-btn.jpg`in the `resources` folder of this repos (Currently getting a 404)

Reply:
Nice !